frontend/mr-admin-flate/src/components/skjema/Datovelger.tsx

```diff
@@ -23,28 +23,34 @@ export function Datovelger<T>({
   const { field: sluttDato } = useController<inferredSchema, "sluttDato">({
     name: "sluttDato",
   });
+
+  const pastDate = () => {
+    const newDate = new Date();
+    const yearsAgo = newDate.setFullYear(newDate.getFullYear() - 3);
```

Comment:
Flytte 3 til en variabel med mer beskrivende navn. Akkurat nå virker den litt "random" og magisk for de som ikke kjenner til hvorfor tallet 3 er valgt. Hva med `offsetAntallAar`?